cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -835,6 +835,14 @@
     </description>
   </property>
 
+  <property>
+    <name>log.kafka.topic.prefix</name>
+    <value>logs.user-v2</value>
+    <description>
+      Kafka log topic to publish logs
+    </description>
+  </property>
+
```

Comment:
These properties are inserted in alphabetical order in each section. So, this one should be earlier.

Is that default value correct? Just wondering...I would suggest for the description:
```
Kafka topic prefix used to publish logs